### ISSUE TITLE
Problems with the control in Jet Set Radio Future

### ISSUE THREAD

**Issue by CyanWolfBR** [Cxbx-Reloaded/game-compatibility#857]
I have been using the emulator for some time, I have been able to play normally, until today, for some reason the controls no longer respond, I make the controls right, I restarted the emulator several times but nothing worked, I can not play Jet Set Radio Future now.
The game usually enters, runs at 60 FPS, audio ok, graphics in the title ok, only the controls that no longer respond.
I use Cxbx - Reloaded db8cf258 (Sep 7 2018).
Sorry for my english, I used the translator as an aid.

**Comment by literalmente-game** [comment]
Please, don't open a new issue before searching for the name of the game you've tested (there is already an open issue for Jet Set Radio). You're always welcome to add any new information to the open issues related to the game here:
- https://github.com/Cxbx-Reloaded/game-compatibility/issues/703 (PAL)
- https://github.com/Cxbx-Reloaded/game-compatibility/issues/409 (NTSC)
After adding the new information to one of the listed issues above, you can close this issue.

**Comment by HyperScorpio** [comment]
%APPDATA% Remove all the data less the save games, check regedit and remove CXBX-Reloaded Folder, run your controler if you have a button, run a new Version and don´t Config anything.
